controller/contribution/revision.php

```diff
@@ -295,9 +298,14 @@ public function add($contrib_type, $contrib)
 			$this->cancel();
 		}
 
+		$this->skip_epv = $this->auth->acl_get('u_titania_mod_extension_validate') && (boolean) $this->request->variable('skip_epv', 0);
+		$can_skip_epv = $this->auth->acl_get('u_titania_mod_extension_validate') && $this->contrib->type instanceof \phpbb\titania\contribution\extension\type;
```

Comment:
The call to `request->variable` can just have the second parameter as `false` and then the casting isn't needed.